Gemfile

```diff
@@ -17,9 +17,9 @@ gem 'simplecov'
 gem 'resque'
 gem 'sidekiq'
 
-group :development, :test do
-  gem 'guard-rspec',                require: false
-  gem 'pry-byebug',                 require: false
-  gem 'rspec'
-  gem 'appraisal'
-end
+gem 'guard-rspec',                require: false
+gem 'pry',                        require: false
+# gem 'pry-byebug',                 require: false
```

Comment:
Ah, I meant to remove it altogether. `byebug` is causing intermittent issues, and I'm not sure why we introduced it in the first place.